lib/translator.js

```diff
@@ -11,9 +11,11 @@ var TranslatedFile = require('./translated-file');
 var PluginError = gutil.PluginError;
 
 
+
 function Translator(options) {
   this.options = lodash.defaults(options, {
       localeDir: 'locale',
+      msgDirs: ['locale'],
```

Comment:
> I saw localeDir as a sort of canonical dir of languages

I like this idea, it can still be done.  The order of the locale directories will always be important (if a msgid exists in multiple catalogs you want the first encountered to win).  We can do the same with the language list, the first locale dir in the list decides which languages we prefer.

If we end up having a lot of translated packages, we could make configuration super simple:
```
localeDirs: ['locale', 'node_modules/**/locale']
```